cli/__init__.py

```diff
@@ -91,31 +93,43 @@ def _quick_ac_lookup():
             exit()
 
 
-def _is_latest_version(suppress_on_error=True):
+def _parse_latest_release_version(resp):
+    # credit: https://stackoverflow.com/a/34366589
+    import json
+
+    latest_release_ver = parse('0')
```

Comment:
what does this do?

Reply:
In case there is no release version, e.g. Now only has devel versions.Then this will be used as a fallback of sorts

Reply:
I there are no release versions I think it should not work

Reply:
With this solutions, if there is no release version and this fallback kicks in, nothing will happen, so I  think that is what we want (the user will not be alerted). Without this, things can break.